<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WFDialog</class>
 <widget class="QDialog" name="WFDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lat_input">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Lattitude</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lon_input">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Longitude</string>
   </property>
  </widget>
  <widget class="QgsMapLayerComboBox" name="coord_layer_combo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Select Coordinate Layer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rotor_diameter_input">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="grid_spacing_input">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Reccomended: 10</string>
   </property>
  </widget>
  <widget class="QLabel" name="Spacing_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Grid Spacing (m):</string>
   </property>
  </widget>
  <widget class="QLabel" name="D_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotor Diameter (m):</string>
   </property>
  </widget>
  <widget class="QLabel" name="Cord_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cordinates:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="use_point_layer_chk">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Point Layer</string>
   </property>
  </widget>
  <widget class="QgsMapLayerComboBox" name="construction_layer_combo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Select Polygon Layer</string>
   </property>
  </widget>
  <widget class="QLabel" name="PolygonInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Construction Areas:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="export_wakes_chk">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Turbine Wakes</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress_bar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>441</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QSlider" name="itterations_slider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Itterations</string>
   </property>
  </widget>
  <widget class="QPushButton" name="init_wind_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Initialise Wind Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="genrate_layout_btn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Layout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plot_wind_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot Wind Rose</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WFDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WFDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
